integration/sawtooth_integration/tests/test_dynamic_network.py

```diff
@@ -181,8 +181,7 @@ def start_node(self, num,
                 raise subprocess.CalledProcessError(proc.pid, proc.returncode)
 
         self.nodes[num] = processes
-        self.clients[num] = IntkeyClient(
-            NodeController.http_address(num), WAIT)
+        self.clients[num] = IntkeyClient(NodeController.http_address(num))
```

Comment:
You should update the IntkeyClient to take a WAIT parameter instead of removing it. It should use that value as a timeout when it submits batches to the rest api, similar to how it is used in the `_post_batches()` functions.